src/Orckestra.Composer/ExceptionFilters/AggregatedComposerExceptionFilter.cs

```diff
@@ -35,16 +36,24 @@ protected override List<ErrorViewModel> GetLocalizedErrors(HttpActionExecutedCon
             }
 
             var query = from exception in innerExceptions.OfType<ComposerException>()
-                        from error in exception.Errors
-                        let localizedErrorMessage = LocalizationProvider.GetLocalizedErrorMessage(error.ErrorCode, Context.CultureInfo)
-                        select new ErrorViewModel
-                        {
-                            ErrorCode = error.ErrorCode,
-                            ErrorMessage = error.ErrorMessage,
-                            LocalizedErrorMessage = localizedErrorMessage
-                        };
+                from error in exception.Errors
+                let localizedErrorMessage =
+                    LocalizationProvider.GetLocalizedErrorMessage(error.ErrorCode, Context.CultureInfo)
+                select ExtendErrorViewModel(error, localizedErrorMessage);
 
             return query.ToList();
         }
+
+        private ErrorViewModel ExtendErrorViewModel(ErrorViewModel error, string localizedErrorMessage)
```

Comment:
rename to CreateErrorViewModel(error) and calculate localizedErrorMessage  inside